venues/auai.org/UAI/2018/webfield/recruitSPCWebfield.html

```diff
@@ -0,0 +1,73 @@
+<html>
```

Comment:
shouldn't webfields be js files? using the code that @zbialecki wrote in OpenReview?

Reply:
Yeah, this one could easily be re-written with the functions from the webfield module.

Reply:
Great, we should do that and don't propagate old code.